lib/cdo/contact_rollups.rb

```diff
@@ -43,8 +43,15 @@ class ContactRollups
   # need to filter here to known courses in the Pardot schema - we can't blindly pass values through
   COURSE_LIST = "'CS Fundamentals','CS in Algebra','CS in Science','CS Principles','Exploring Computer Science','CS Discoveries'".freeze
 
-  # Values of forms.kind field we care about
-  FORM_KINDS = %w(BringToSchool2013 CSEdWeekEvent2013 DistrictPartnerSubmission HelpUs2013 Petition K5OnlineProfessionalDevelopmentPostSurvey).freeze
+  # Values of forms.kind field with form data we care about
+  FORM_KINDS_WITH_DATA = %w(BringToSchool2013 CSEdWeekEvent2013 DistrictPartnerSubmission HelpUs2013 Petition K5OnlineProfessionalDevelopmentPostSurvey).freeze
+
+  # Kinds of forms that indicate this contact is a teacher
+  FORM_LIST_TEACHER = "'BringToSchool2013','ClassSubmission','DistrictPartnerSubmission',
+      'PLP interest form','Teacher interest form','School interest form','HelpUs2013',
+      'K5OnlineProfessionalDevelopmentPostSurvey','K5ProfessionalDevelopmentSurvey',
+      'ProfessionalDevelopmentWorkshop','ProfessionalDevelopmentWorkshopSignup',
+      'StudentNomination','TeacherNomination'"
```

Comment:
Thx, have split the line and taken your rename suggestion. Will consider changing to string array in the future. (Since it is only used in one place and would have to rejoin it into this form for the one place it is used, not sure the dynamic reassembly is a net win for code complexity.)